ocaml/xapi/xapi_globs.ml

```diff
@@ -972,6 +972,8 @@ let pvsproxy_close_cache_vdi = ref "/opt/citrix/pvsproxy/close-cache-vdi.sh"
 
 let yum_repos_config_dir = ref "/etc/yum.repos.d"
 
+let dnf_repo_config_file = ref "/etc/dnf/repos.override.d/99-config_manager.repo"
```

Comment:
Hi @DeliZhangX 
May I please know why the password will be written into this file?
It is done by `dnf` automatically? Its predecessor `yum` will write into the same repo config file. So the password will be removed with the  `write_initial_yum_config` in the final block as the comment describes.

Reply:
Shouldn't it be with the repo name like `remote-<UUID>.repo`? Why now it is `/etc/dnf/repos.override.d/99-config_manager.repo`?

Reply:
This is the dnf5's behaviour.